lib/Github/Api/PullRequest/Comments.php

```diff
@@ -11,9 +11,13 @@
  */
 class Comments extends AbstractApi
 {
-    public function all($username, $repository, $pullRequest)
+    public function all($username, $repository, $pullRequest = null)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($pullRequest).'/comments');
+        if (null !== $pullRequest) {
+            return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($pullRequest).'/comments');
+        }
+
+        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/comments');
```

Comment:
Actually, it's there to allow https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository which is not directly related to the reviews, but needed if you want to fully leverage the reviews API. But I can move it to its own PR nonetheless. What do you think?